lib/recurly/base.php

```diff
@@ -194,8 +194,10 @@ public function getLinks() {
     'plan_code' => 'string',
     'plan_codes' => 'array',
     'pending_subscription' => 'Recurly_Subscription',
+    'risk_rules_triggered' => 'array',
     'redemption' => 'Recurly_CouponRedemption',
     'redemptions' => 'Recurly_CouponRedemptionList',
+    'rule' => 'Recurly_FraudRiskRule',
```

Comment:
Perhaps there is a way to just parse it into an array with keys. I could not find a way to do it but we could maybe add it.